index.xml

```diff
@@ -13402,7 +13420,9 @@ C implements A;</x:codeblock>
                     <a class='dfnref' href='#dfn-extended-attribute'>extended attribute</a>-->, then set
                     <var>desc</var>.<span class='prop'>[[Writable]]</span> to <span class='esvalue'>true</span>, otherwise set it to
                     <span class='esvalue'>false</span>.</li>
-                  <li>If the named property is defined to be <a class='dfnref' href='#dfn-unenumerable'>unenumerable</a>,
+                  <li>If <var>O</var> implements an <a class='dfnref' href='#dfn-interface'>interface</a> with the
+                    <a class='xattr' href='#LegacyUnenumerableNamedProperties'>[[LegacyUnenumerableNamedProperties]]</a>
```

Comment:
`[LegacyUnenumerableProperties]`, not `[[LegacyUnenumerableProperties]].

This phrasing is pretty weird, in that it allows `[LegacyUnenumerableProperties]` to appear on one interface but the named getter to appear on another interface.  It probably doesn't matter in practice, of course...